pkg/sdn/plugin/runtime.go

```diff
@@ -0,0 +1,55 @@
+package plugin
+
+import (
+	"fmt"
+	"time"
+
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+	kubeletapi "k8s.io/kubernetes/pkg/kubelet/api"
+	kruntimeapi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+	kubeletremote "k8s.io/kubernetes/pkg/kubelet/remote"
+)
+
+func (node *OsdnNode) getRuntimeService() (kubeletapi.RuntimeService, error) {
+	if node.runtimeService != nil {
+		return node.runtimeService, nil
+	}
+
+	// Kubelet starts asynchronously and when we get an Update op, kubelet may not have created runtime endpoint.
+	// So try couple of times before bailing out (~5 second timeout).
```

Comment:
You should wait longer.   Heavily contended systems could flake because of this.